benchmarks/yamauchi_takei_2016_anelasticity/anelasticity_temperature.cc

```diff
@@ -298,15 +298,15 @@ namespace aspect
                                                   /background_slope)+((std::sqrt(2*M_PI)/2)*peak_amplitude*peak_width*(1-
                                                                       std::erf((std::log(peak_period/normalised_period))/(std::sqrt(2)*peak_width)))));
       // determine J2 term (imaginary part of complex compliance)
-      loss_compliance=unrelaxed_compliance*(M_PI/2)*(background_amplitude*(std::pow(normalised_period,background_slope))+\
-                                                     (peak_amplitude*std::exp(-1*(std::pow(std::log(peak_period/normalised_period),2)
-                                                                                  /(2*std::pow(peak_width,2))))))+(unrelaxed_compliance*normalised_period);
+      //double loss_compliance=unrelaxed_compliance*(M_PI/2)*(background_amplitude*(std::pow(normalised_period,background_slope))+
+      // (peak_amplitude*std::exp(-1*(std::pow(std::log(peak_period/normalised_period),2)/
+      // (2*std::pow(peak_width,2))))))+(unrelaxed_compliance*normalised_period);
       // calculate anharmonic Vs
-      anharmonic_Vs=1/(std::sqrt(density*unrelaxed_compliance)*1e3);
+      // anharmonic_Vs=1/(std::sqrt(density*unrelaxed_compliance)*1e3);
       // calculate Vs
       anelastic_Vs=1/(std::sqrt(density*storage_compliance)*1e3);
       // calculate attenuation
-      attenuation=loss_compliance/storage_compliance;
+      // attenuation=loss_compliance/storage_compliance;
```

Comment:
Should the commented lines above just be removed?

Reply:
I am not sure if it makes sense to preserve the logic for computing all these quantities. I can remove all of it, if you prefer.

Reply:
I'm not familiar with this benchmark either, so maybe best to presence the logic for the time being.